### DISCUSSION TITLE
Adding Sub-Properties to Composites via API in Style Manager

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5906] [Q&A] [answered]
How can I add a property within a composite in the Style Manager using an API? Is there any possibility to do this? I know the addProperty method works, but since a composite is also a property, I want to add a sub-child property to it.

@artf

**Comment by artf** [comment, marked as answer]
You should be able to access sub-properties and add a new one
```js
compositeProperty.properties.add({ type: 'number', ... })
```
